tests/basetest.rb

```diff
@@ -85,20 +85,56 @@ def password
     self.class.password
   end
 
+  def self.telnet_port
```

Comment:
Why are you exposing a telnet port?  Are you using a non-default port in your test environment?

Reply:
Yea I am using non-default in my local test environment